src/nvim/ui.c

```diff
@@ -186,6 +186,10 @@ Dictionary hlattrs2dict(const HlAttrs *aep, bool use_rgb)
     PUT(hl, "bold", BOOLEAN_OBJ(true));
   }
 
+  if (mask & HL_STANDOUT) {
```

Comment:
Also remove `HL_STANDOUT` from "reverse" condition below.

Reply:
Oh yes, I didn't see it, thanks for pointing out